.env.development

```diff
@@ -1,5 +1,13 @@
+GITLAB_SECRET=GITLAB_SECRET
+GITLAB_TOKEN=GITLAB_TOKEN
 NODE_ENV=development
-POSTGRES_HOST=0.0.0.0
+POSTGRES_HOST=localhost
 POSTGRES_PASSWORD=root
 POSTGRES_PORT=54320
 POSTGRES_USER=root
+POSTGRES_DATABASE_NAME=homer
+SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN=SLACK_BOT_USER_O_AUTH_ACCESS_TOKEN
+SLACK_SIGNING_SECRET=SLACK_SIGNING_SECRET
+EMAIL_DOMAINS=my-domain.com,ext.my-domain.com
+GITLAB_URL=https://my-git.domain.com
+TICKET_MANAGEMENT_URL_PATTERN=https://my-ticket-management.com/view/{ticketId}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.

Reply:
Fixed